### PR TITLE
Add CTUImportCppThreshold for C++ files

### PR DESCRIPTION
**Pull request by martong**

### CODE REVIEW

File: clang/include/clang/StaticAnalyzer/Core/AnalyzerOptions.def [hunk 1]
```diff
@@ -317,9 +317,16 @@ ANALYZER_OPTION(bool, ShouldEmitFixItHintsAsRemarks, "fixits-as-remarks",
...
                 "analysis with many interdependent definitions located in "
-                "various translation units.",
+                "various translation units. This is valid only for non C++ "
+                "source files.",
+                24u)
+
+ANALYZER_OPTION(unsigned, CTUImportCppThreshold, "ctu-import-cpp-threshold",
+                "The maximal amount of translation units that is considered "
+                "for import when inlining functions during CTU analysis of C++ "
+                "source files. ",
```
Comment (gamesh411) [inline clang/include/clang/StaticAnalyzer/Core/AnalyzerOptions.def:329]:
```suggestion
                "source files.",
```

**Comment by gamesh411** [comment]
Please remove the whitespace, otherwise feel free to commit anytime. LGTM.

**Comment by martong** [comment]
same as https://reviews.llvm.org/D83475

**Comment by martong** [comment]
Ups:

```
[==========] Running 1 test from 1 test case.
[----------] Global test environment set-up.
[----------] 1 test from CrossTranslationUnit
[ RUN      ] CrossTranslationUnit.RespectsLoadThreshold
../../git/llvm-project/clang/unittests/CrossTU/CrossTranslationUnitTest.cpp:171: Failure
Value of: Success
  Actual: true
Expected: false
[  FAILED  ] CrossTranslationUnit.RespectsLoadThreshold (26 ms)
[----------] 1 test from CrossTranslationUnit (26 ms total)

```

**Pull request merged**